Tasks/Maven/task.loc.json

```diff
@@ -16,7 +16,7 @@
   "author": "Microsoft Corporation",
   "version": {
     "Major": 1,
-    "Minor": 127,
+    "Minor": 128,
```

Comment:
All the other tasks were already updated to minor version 128. I'm not sure how this loc file was missed.